tasks/translation-ollama/model.py

```diff
@@ -0,0 +1,69 @@
+from typing import Optional
+
+import logging
+import subprocess
+import time
+
+from ollama import ResponseError
+
+import requests
+import ollama
+
+
+OLLAMA_SERVER_URL = "http://127.0.0.1:11434"
+LLM = "llama3"
+
+
+def warmup() -> None:
+    ollama.chat(
+        model = LLM,
+        messages = [{
+            "role": "user",
+            "content": f"This is a test and a warmup. Respond with \"OK\"."
+        }]
+    )
+
+
+def isOllamaInstalled() -> bool:
+    try:
+        subprocess.run(["ollama", "--version"], check = True, stdout = subprocess.PIPE, stderr = subprocess.PIPE)
+        return True
+    except (subprocess.CalledProcessError, FileNotFoundError):
+        return False
+
+
+def installOllama() -> None:
+    try:
+        subprocess.run("curl https://ollama.ai/install.sh | sh", shell = True, check = True)
+        logging.info(">> [OllamaRAG] Ollama installation was successful")
+    except subprocess.CalledProcessError as e:
+        raise RuntimeError(f">> [OllamaRAG] An error occurred during Ollama installation: {e}")
+
+
+def isOllamaServerActiveAndRunning() -> bool:
+    try:
+        response = requests.get(OLLAMA_SERVER_URL)
+        return bool(response.status_code == 200)
```

Comment:
You can directly return the value of `response.ok` which is a boolean property designating if the request failed or not